src/Symfony/Component/Security/Http/Firewall/ContextListener.php

```diff
@@ -45,11 +48,16 @@ class ContextListener implements ListenerInterface
     private $trustResolver;
     private $logoutOnUserChange = false;
 
+    /**
+     * @var AbstractRememberMeServices
```

Comment:
this annotation is inconsistent with the signature - it should be removed altogether I think